reactor-netty-core/src/test/java/reactor/netty/resources/ConnectionProviderTest.java

```diff
@@ -127,4 +127,4 @@ public int permitMaximum() {
 		public void returnPermits(int returned) {
 		}
 	}
-}
+}
```

Comment:
nit: the newline has been removed at the end of the file, can you just re-add it, so the ConnectionProviderTest won't appear in the list of changes ?

Reply:
absolutely :), sorry.